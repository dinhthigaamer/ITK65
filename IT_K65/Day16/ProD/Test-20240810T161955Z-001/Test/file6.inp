930
72452893 -904718603 491843550 290006657 734239706 -300485447 603855613 -104567772 -202489833 102837923 494427367 840225556 569697539 180048123 -268040826 -619554211 -819907859 580473372 88082312 301807987 992383310 322565666 -552461327 -758517159 -190507664 891761539 -40708242 138488377 194340385 8607759 -243670641 122531872 324112604 -644890821 498274335 -217785506 503335141 -912951463 -840309218 -122169158 -927369916 317001107 -906905775 -191958004 -562502250 622044244 -780850041 349774573 180798643 -481768983 -736722771 131603230 -400979622 -342866634 10521218 788217690 124702390 -540336430 -308335712 379975182 418824774 -950582349 297067734 648863680 223051464 406672677 -587072936 -882789572 -354256877 335679243 -750130100 -185474533 -362780644 -462051806 484955401 -198930445 412406631 879754889 -312547163 452239449 -742265726 -961865082 -282087500 -596180914 53611041 -924937282 -450254573 475419553 621560860 -343857851 410979931 -880218918 -333615092 193644371 -3724082 -751842034 -869556663 68650172 -49590477 -26155484 -44641240 -141978297 -641104306 -209990656 -927014014 -726320738 -533506257 455824590 -775179088 631683023 -974858199 -157471745 -629658630 71825477 -6235611 589501123 75350453 -951606392 968137983 396084807 154566181 804898798 148302932 -746790329 -62401301 17931575 -397758895 -538241383 843380168 100424785 -139413669 -693192005 661748870 412094007 636849111 -942653909 -490725406 -513586118 545883870 -418271206 853135897 -554185351 -266436302 -684110305 300516412 508362608 110839114 -245863229 785328556 460919553 -600410145 -45207278 180477653 594033090 951915404 230762595 818836128 -820032894 -325630566 -26757781 559831062 434057449 449072970 811296824 29321543 482889146 611281316 -608657475 -714206080 -173059176 -326778710 -793259056 -655703493 -554662145 -838115940 556943539 -772995537 640012629 63545593 -8507443 -36391691 574654942 -51292176 434692199 -160734018 114140869 988846589 -8123790 -841563682 -48828838 417574497 -152765543 135601889 203418748 -481030874 170390081 434753888 551663904 -47029005 -897050276 660850870 272358942 535033090 576083300 639017232 -900613268 -303521963 -687909074 583551863 -720943923 650533716 812096420 -77537003 -164150750 205038620 -26862427 393385834 -160285547 75883702 -25093494 350684908 362442649 -437827629 -66214209 524282478 -153531855 -377675831 390019759 -302866959 308833005 936635969 -90491213 965363849 541998148 514551654 -139330413 -541193691 15050017 -297588987 876308131 72708982 509733375 781356880 818462649 -438949302 -852386951 -989690363 488565351 -871294434 -343950620 -27838149 451290143 -898849519 -237213894 -751554452 -404930485 790964783 815030213 91029873 446773541 77567664 -638884114 -477358063 -551623323 -742482779 -804598878 -222986742 309761824 925377744 303453964 -582289396 987038920 134901659 -373114459 510567746 824545660 747365761 202797830 920927218 -779874158 500514564 775721914 577925714 86631652 -80125970 413955777 943819829 479136811 -961478696 857094674 -377914514 -816543738 17692286 -722696418 -694404240 -676211363 -62984521 341509500 -944831140 -692054818 844695715 -598886102 -195862533 -893269172 -572570609 -831728480 -696305252 -814177782 940313270 -102341125 -985950340 669155541 -720755155 225736619 123516094 -355756143 204207271 -681888926 755493462 -584707480 627131181 -889280704 -129064150 -529135248 309506940 690875279 -812309131 -29418650 313982494 -812602976 -202348327 783919799 749497192 48486264 558257496 992417323 -322336573 162771828 714852024 -659997896 -107433426 -272195929 170064700 582751868 36666512 -733367187 677694533 -875021809 641292298 -456567886 -414402407 -275794803 601333833 -769454209 -510961645 -765406515 910147321 -472663549 325598466 -313540288 -587274107 -623777374 -943459131 636855024 -163660088 -441332546 320248904 -825427401 685506411 -696257259 -322895771 767730949 486388626 75567183 109010387 -36427593 888200626 851839672 -165999077 -976582158 -794855305 -525114218 -515488569 -152341501 830188356 -425733767 381547731 650687130 353112127 -58438125 610898567 827969159 -242212636 -864460803 660117471 -916337305 176802698 -404907461 698019612 -450771662 -923912007 375387045 -573581221 -173433892 -303971341 -384402306 -403213246 -599897183 988755690 -876951515 -510039620 310373476 -110471375 -693334748 -589447086 629318326 -52004056 303200205 973377160 -619892155 410653239 -625094368 466444934 -385731717 398262218 -279296246 191137862 -929244250 -308184703 -381598811 -665022539 -269461088 482245290 311161739 415842967 -776202160 784983965 -741358185 982351923 -479600828 288085722 -216187437 -670816623 434795122 601436578 -617884580 79114497 74177969 176957402 -126630451 253136191 879334311 4227842 -69901282 -355701992 384619691 564155947 114288498 -595618450 161080670 -264357751 365916584 -976693027 462877324 -808238591 -277798291 -257914867 776013090 135298648 737153138 504436245 891609595 193450717 450478459 -81925505 768059169 -692717604 -992921541 623705061 208068363 -774340903 243083040 154138331 429021092 -784564525 31338830 -170824535 750709503 586797146 710878682 433418835 -298915789 -740277069 750813219 606466021 125780395 -885754691 560257664 -543287991 713238958 342772484 -890334688 -746600542 -298092524 -713857585 -180022053 -680519972 -548050177 924678942 994076753 -415731156 -283101132 265660895 -357167651 -161823773 -616920864 -447044009 -278628272 -291182148 713101336 352790303 -430863601 749468174 208521077 -831838728 751509588 -254533851 -644743202 -571478034 260442694 -610249901 707535031 -976274921 -802573114 132802121 -378819202 -810047314 -387604080 970391985 -518494326 314700696 80188664 -175637660 336437304 -370871211 949700696 -766136158 40000558 475604936 208184981 -412919821 348360370 -57493379 981651973 583764457 -471770825 585472442 -768530706 346684421 300536208 -324198178 888647717 -370439275 -359218152 192126838 -556290629 -913112860 -990169030 196989493 354635698 49253369 -391868822 123486006 -645568730 -172514283 -855445712 -2380723 -923463686 435358120 -325211918 239565057 -481424844 -479229303 980637354 -804420875 -234159960 745379502 -746185012 952533415 870830202 828918958 -637822472 -706325082 -31663404 189106826 503579477 -629028559 -673946234 -190579621 -400558667 549463210 -125910788 857841565 869790499 -394178017 -799048689 -756973388 -410437158 -101141334 211712751 -81755326 -421042460 -645031442 608837407 483844769 9829542 135646609 -104690509 520128875 -714596346 -357123463 -43137370 -500828357 -254512576 336496798 -809593020 834374873 -287715464 -670703195 217280708 -980202825 -771558399 -414440128 -895163097 238749044 -392484973 -223690322 419945863 785571336 54895436 -683215671 -94755558 -752177620 -677488092 -887777445 -171177550 -345230099 -111300111 7432685 -51905866 -206999284 -828042361 697199657 907235180 -937054906 -508106242 150419505 186405837 147661891 115219578 -86814093 807378027 -338684953 597188469 -268633553 33619780 428850693 -451750053 181223122 798069748 -643812390 -973373262 -30377461 -845383422 692009286 -925684548 -293657832 325891365 -940888098 759674053 -515661310 632565544 52519648 -420132545 274386154 420362918 -651427757 -223770859 722544384 408618437 740124035 -526870065 -538118304 -391974929 -156519389 428291094 606058050 300231399 -994799427 118031132 626563891 -347533185 223657877 387571679 923182241 -870539734 276623636 236851326 -377760886 956319851 -323120909 -363782926 528906086 300586434 383481843 111808880 -263887616 60191733 186892694 1476615 727186181 -497140491 -453038076 -278471918 816955016 -241393080 934825208 -60230396 -730041796 -973310282 -582406795 868423981 -370985900 786854448 -223617343 280199874 -930213798 566945312 23652516 664542785 310157076 755726164 684684071 388177264 402540388 424798521 -374750647 924309500 397521437 844101134 404610845 -633367807 -612855820 472338040 115931753 -432049314 -966461556 -239216692 508464336 916221222 216330028 -517448223 -744633625 -761401972 656109308 72836716 -772924706 -202246100 -165871667 -991426281 -490838183 209351416 134956137 -36128588 -754413937 -616596290 696201705 -201191199 967464971 -244181984 -729907974 -788184745 230837860 243372645 124496535 267599477 -21203560 -783905757 -762885763 -240509055 191975130 -923478327 617734403 -123196318 -633355521 343206227 -369228912 380050129 583366808 -616481273 -104857026 -329439415 -744504693 616895 973731895 -260120634 -18702722 124543287 815116785 -442359253 -8453652 -776635259 943556445 307146897 -746832078 460445488 331161585 515936801 -891485564 -307506856 -25141822 264467245 405294148 955481117 -823215707 -978526793 -62818433 -986719237 -715124024 -660951957 -467478555 112026226 -633149674 -538377167 -712517486 -608920415 -434475732 444722179 -161708946 99870253 -44110034 376218507 -513043935 220010296 -75059089 663132708 846754796 2793266 416712412 545786451 262898403 442511997 -441241260 -479587986 -752123290 720443264 500646237 737842681 -571966572 -798463881 -477248830 89875164 410339460 -359022689 -251280099 -441455542 -538431186 -406793869 374561004 -108755237 -440143410 -414643200 143867946 -49941076 -636997997 547159417 575916829 -633022978 581139521 576315198 -793845391 871347528 -506166840 825701388 930629716 -769786740 202716986 -551588923 -252462830 -372930897 563690828 522489675 -299668615 -339174716 677509330 594499381 -571027567 182577546 559928772 459938516 -174980002 -191778233 130950205 -314987139 -943679693 184114713 -599815301 -99757829 -23146973 -943396301 -478816343 627897495 388165214 -560916643 429415835 -278568751 -519782514 -820609847 -81949754 367306957 155156343 549258693 54150992 -559386278 139180816 -795102543 105661573 41934068 -261150025 -672915777 
72452893 -904718603 491843550 290006657 734239706 -300485447 603855613 -104567772 -202489833 102837923 494427367 840225556 569697539 180048123 -268040826 -619554211 -819907859 580473372 88082312 301807987 992383310 322565666 -552461327 -758517159 -190507664 891761539 -40708242 138488377 194340385 8607759 -243670641 122531872 324112604 -644890821 498274335 -217785506 503335141 -912951463 -840309218 -122169158 -927369916 317001107 -906905775 -191958004 -562502250 622044244 -780850041 349774573 180798643 -481768983 -736722771 131603230 -400979622 -342866634 10521218 788217690 124702390 -540336430 -308335712 379975182 418824774 -950582349 297067734 648863680 223051464 406672677 -587072936 -882789572 -354256877 335679243 -750130100 -185474533 -362780644 -462051806 484955401 -198930445 412406631 879754889 -312547163 452239449 -742265726 -961865082 -282087500 -596180914 53611041 -924937282 -450254573 475419553 621560860 -343857851 410979931 -880218918 -333615092 193644371 -3724082 -751842034 -869556663 68650172 -49590477 -26155484 -44641240 -141978297 -641104306 -209990656 -927014014 -726320738 -533506257 455824590 -775179088 631683023 -974858199 -157471745 -629658630 71825477 -6235611 589501123 75350453 -951606392 968137983 396084807 154566181 804898798 148302932 -746790329 -62401301 17931575 -397758895 -538241383 843380168 100424785 -139413669 -693192005 661748870 412094007 636849111 -942653909 -490725406 -513586118 545883870 -418271206 853135897 -554185351 -266436302 -684110305 300516412 508362608 110839114 -245863229 785328556 460919553 -600410145 -45207278 180477653 594033090 951915404 230762595 818836128 -820032894 -325630566 -26757781 559831062 434057449 449072970 811296824 29321543 482889146 611281316 -608657475 -714206080 -173059176 -326778710 -793259056 -655703493 -554662145 -838115940 556943539 -772995537 640012629 63545593 -8507443 -36391691 574654942 -51292176 434692199 -160734018 114140869 988846589 -8123790 -841563682 -48828838 417574497 -152765543 135601889 203418748 -481030874 170390081 434753888 551663904 -47029005 -897050276 660850870 272358942 535033090 576083300 639017232 -900613268 -303521963 -687909074 583551863 -720943923 650533716 812096420 -77537003 -164150750 205038620 -26862427 393385834 -160285547 75883702 -25093494 350684908 362442649 -437827629 -66214209 524282478 -153531855 -377675831 390019759 -302866959 308833005 936635969 -90491213 965363849 541998148 514551654 -139330413 -541193691 15050017 -297588987 876308131 72708982 509733375 781356880 818462649 -438949302 -852386951 -989690363 488565351 -871294434 -343950620 -27838149 451290143 -898849519 -237213894 -751554452 -404930485 790964783 815030213 91029873 446773541 77567664 -638884114 -477358063 -551623323 -742482779 -804598878 -222986742 309761824 925377744 303453964 -582289396 987038920 134901659 -373114459 510567746 824545660 747365761 202797830 920927218 -779874158 500514564 775721914 577925714 86631652 -80125970 413955777 943819829 479136811 -961478696 857094674 -377914514 -816543738 17692286 -722696418 -694404240 -676211363 -62984521 341509500 -944831140 -692054818 844695715 -598886102 -195862533 -893269172 -572570609 -831728480 -696305252 -814177782 940313270 -102341125 -985950340 669155541 -720755155 225736619 123516094 -355756143 204207271 -681888926 755493462 -584707480 627131181 -889280704 -129064150 -529135248 309506940 690875279 -812309131 -29418650 313982494 -812602976 -202348327 783919799 749497192 48486264 558257496 992417323 -322336573 162771828 714852024 -659997896 -107433426 -272195929 170064700 582751868 36666512 -733367187 677694533 -875021809 641292298 -456567886 -414402407 -275794803 601333833 -769454209 -510961645 -765406515 910147321 -472663549 325598466 -313540288 -587274107 -623777374 -943459131 636855024 -163660088 -441332546 320248904 -825427401 685506411 -696257259 -322895771 767730949 486388626 75567183 109010387 -36427593 888200626 851839672 -165999077 -976582158 -794855305 -525114218 -515488569 -152341501 830188356 -425733767 381547731 650687130 353112127 -58438125 610898567 827969159 -242212636 -864460803 660117471 -916337305 176802698 -404907461 698019612 -450771662 -923912007 375387045 -573581221 -173433892 -303971341 -384402306 -403213246 -599897183 988755690 -876951515 -510039620 310373476 -110471375 -693334748 -589447086 629318326 -52004056 303200205 973377160 -619892155 410653239 -625094368 466444934 -385731717 398262218 -279296246 191137862 -929244250 -308184703 -381598811 -665022539 -269461088 482245290 311161739 415842967 -776202160 784983965 -741358185 982351923 -479600828 288085722 -216187437 -670816623 434795122 601436578 -617884580 79114497 74177969 176957402 -126630451 253136191 879334311 4227842 -69901282 -355701992 384619691 564155947 114288498 -595618450 161080670 -264357751 365916584 -976693027 462877324 -808238591 -277798291 -257914867 776013090 135298648 737153138 504436245 891609595 193450717 450478459 -81925505 768059169 -692717604 -992921541 623705061 208068363 -774340903 243083040 154138331 429021092 -784564525 31338830 -170824535 750709503 586797146 710878682 433418835 -298915789 -740277069 750813219 606466021 125780395 -885754691 560257664 -543287991 713238958 342772484 -890334688 -746600542 -298092524 -713857585 -180022053 -680519972 -548050177 924678942 994076753 -415731156 -283101132 265660895 -357167651 -161823773 -616920864 -447044009 -278628272 -291182148 713101336 352790303 -430863601 749468174 208521077 -831838728 751509588 -254533851 -644743202 -571478034 260442694 -610249901 707535031 -976274921 -802573114 132802121 -378819202 -810047314 -387604080 970391985 -518494326 314700696 80188664 -175637660 336437304 -370871211 949700696 -766136158 40000558 475604936 208184981 -412919821 348360370 -57493379 981651973 583764457 -471770825 585472442 -768530706 346684421 300536208 -324198178 888647717 -370439275 -359218152 192126838 -556290629 -913112860 -990169030 196989493 354635698 49253369 -391868822 123486006 -645568730 -172514283 -855445712 -2380723 -923463686 435358120 -325211918 239565057 -481424844 -479229303 980637354 -804420875 -234159960 745379502 -746185012 952533415 870830202 828918958 -637822472 -706325082 -31663404 189106826 503579477 -629028559 -673946234 -190579621 -400558667 549463210 -125910788 857841565 869790499 -394178017 -799048689 -756973388 -410437158 -101141334 211712751 -81755326 -421042460 -645031442 608837407 483844769 9829542 135646609 -104690509 520128875 -714596346 -357123463 -43137370 -500828357 -254512576 336496798 -809593020 834374873 -287715464 -670703195 217280708 -980202825 -771558399 -414440128 -895163097 238749044 -392484973 -223690322 419945863 785571336 54895436 -683215671 -94755558 -752177620 -677488092 -887777445 -171177550 -345230099 -111300111 7432685 -51905866 -206999284 -828042361 697199657 907235180 -937054906 -508106242 150419505 186405837 147661891 115219578 -86814093 807378027 -338684953 597188469 -268633553 33619780 428850693 -451750053 181223122 798069748 -643812390 -973373262 -30377461 -845383422 692009286 -925684548 -293657832 325891365 -940888098 759674053 -515661310 632565544 52519648 -420132545 274386154 420362918 -651427757 -223770859 722544384 408618437 740124035 -526870065 -538118304 -391974929 -156519389 428291094 606058050 300231399 -994799427 118031132 626563891 -347533185 223657877 387571679 923182241 -870539734 276623636 236851326 -377760886 956319851 -323120909 -363782926 528906086 300586434 383481843 111808880 -263887616 60191733 186892694 1476615 727186181 -497140491 -453038076 -278471918 816955016 -241393080 934825208 -60230396 -730041796 -973310282 -582406795 868423981 -370985900 786854448 -223617343 280199874 -930213798 566945312 23652516 664542785 310157076 755726164 684684071 388177264 402540388 424798521 -374750647 924309500 397521437 844101134 404610845 -633367807 -612855820 472338040 115931753 -432049314 -966461556 -239216692 508464336 916221222 216330028 -517448223 -744633625 -761401972 656109308 72836716 -772924706 -202246100 -165871667 -991426281 -490838183 209351416 134956137 -36128588 -754413937 -616596290 696201705 -201191199 967464971 -244181984 -729907974 -788184745 230837860 243372645 124496535 267599477 -21203560 -783905757 -762885763 -240509055 191975130 -923478327 617734403 -123196318 -633355521 343206227 -369228912 380050129 583366808 -616481273 -104857026 -329439415 -744504693 616895 973731895 -260120634 -18702722 124543287 815116785 -442359253 -8453652 -776635259 943556445 307146897 -746832078 460445488 331161585 515936801 -891485564 -307506856 -25141822 264467245 405294148 955481117 -823215707 -978526793 -62818433 -986719237 -715124024 -660951957 -467478555 112026226 -633149674 -538377167 -712517486 -608920415 -434475732 444722179 -161708946 99870253 -44110034 376218507 -513043935 220010296 -75059089 663132708 846754796 2793266 416712412 545786451 262898403 442511997 -441241260 -479587986 -752123290 720443264 500646237 737842681 -571966572 -798463881 -477248830 89875164 410339460 -359022689 -251280099 788770493 -538431186 -406793869 374561004 -108755237 -440143410 -414643200 143867946 -49941076 -636997997 547159417 575916829 -633022978 581139521 576315198 -793845391 871347528 -506166840 825701388 930629716 -769786740 202716986 -551588923 -252462830 -372930897 563690828 522489675 -299668615 -339174716 677509330 594499381 -571027567 182577546 559928772 459938516 -174980002 -191778233 130950205 -314987139 -943679693 184114713 -599815301 -99757829 -23146973 -943396301 -478816343 627897495 388165214 -560916643 429415835 -278568751 -519782514 -820609847 -81949754 367306957 155156343 549258693 54150992 -559386278 139180816 -795102543 105661573 41934068 -261150025 -672915777 