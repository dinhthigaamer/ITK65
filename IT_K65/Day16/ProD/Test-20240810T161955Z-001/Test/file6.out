-33130845028306
